docs/src/techpapers/netcdf_io.rst

```diff
@@ -0,0 +1,116 @@
+.. _netcdf_io:
+
+.. testsetup:: chunk_control
+
+    import iris
+    from iris.fileformats.netcdf.loader import CHUNK_CONTROL
+
+    from pathlib import Path
+    import shutil
+    import tempfile
+
+    tmp_dir = Path(tempfile.mkdtemp())
+    tmp_filepath = tmp_dir / "tmp.nc"
+
+    cube = iris.load(iris.sample_data_path("E1_north_america.nc"))[0]
+    iris.save(cube, tmp_filepath, chunksizes=(120, 37, 49))
+
+
+.. testcleanup:: chunk_control
+
+    shutil.rmtree(tmp_dir)
+
+
+=============================
+NetCDF I/O Handling in Iris
+=============================
+
+This document provides a basic account of how NetCDF files work within Iris.
+
+This document is still a work in progress, so might include blank or unfinished sections,
+watch this space!
```

Comment:
This would be good as a [RST admonition](https://docutils.sourceforge.io/docs/ref/rst/directives.html#admonitions) - to draw attention to it. You can pick which type.

An example:

![image](https://github.com/SciTools/iris/assets/40734014/3cbe77e1-4039-44b2-a4b2-2d34305ca69c)